lib/fathead/apple_discussions/parse.py

```diff
@@ -43,7 +43,7 @@
 FHTEMPLATE = """\
 <p><b>Answered by {username} ({date})</b></p>
 {information}
-"""
+""".replace("\r\n", "").replace("\n", "").replace("\\n", "").replace(r"\x", "\\x")
```

Comment:
Somehow this still missed a few `^M`.

Perhaps we also need `.replace("\r", "")`?

Reply:
We also need to replace `\x` with `\\x`